crates/router/src/configs/settings.rs

```diff
@@ -723,6 +724,12 @@ pub struct ApplePayDecryptConfig {
     pub apple_pay_merchant_cert_key: Secret<String>,
 }
 
+#[derive(Debug, Deserialize, Clone, Default)]
+pub struct PazeDecryptConfig {
+    pub paze_private_key: Secret<String>,
+    pub paze_private_key_passphrase: Secret<String>,
+}
```

Comment:
Once we make the `paze_decrypt_keys` field optional, then we can validate that these values are not empty, if provided.

Reply:
Actually PAZE will offer universal keys which will be used by all the merchants hence this need not be optional because we will always have the Paze configs.

Reply:
Validation added